src/redux/products/thunks.js

```diff
@@ -89,13 +89,10 @@ export const postProducts = (name,description,price,stock) => {
         const json = await response.json();
         if (response.status === 201) {
           dispatch(postProductsSuccess(json.data));
-          console.log('Product added');
         } else {
```

Comment:
dont leave else statement empty, you should dispatch products error here too, because any other response than 201 wouldn't succesfully create a product on the DB

Reply:
done